collectors/osidb_scheduler/apps.py

```diff
@@ -0,0 +1,11 @@
+"""
+    OSIDB service
+"""
+
+from django.apps import AppConfig
+
+
+class OSIDBScheduler(AppConfig):
+    """OSIDB scheduler service"""
+
+    name = "collectors.osidb_scheduler"
```

Comment:
I do not think that `collectors` is the right place. The collectors are meant to bring some data to OSIDB from external sources which does not happen here. I assume that this module should run rather internal tasks like housekeeping here or some checks but not really a data collection.

Reply:
I've removed the collector and created a periodic celery task instead